content/docs/rfcs/48/README.md

```diff
@@ -0,0 +1,105 @@
+---
+slug: 48
+title: 48/RLN-INTEREP-SPEC
+name: Interep as group management for RLN
+status: raw
+category:  
+tags: rln
+editor: Aaryamann Challani <aaryamann@status.im>
+contributors:
+---
+
+# Abstract
+
+This spec integrates [Interep](https://interep.link) into the [RLN](https://rfc.vac.dev/spec/32/) spec. 
```

Comment:
This is for a separate issue but this should probably refer to RLNP2P (can fix-up later)

Reply:
Also I just realized it is Interep, not Interrep. I always read it as Inter(operable)-rep(utation)

Reply:
Addressed in [76516c1](https://github.com/vacp2p/rfc/pull/564/commits/76516c179bb029bd584257e05a0c34572d756729)